app/services/moves/finder.rb

```diff
@@ -2,15 +2,40 @@
 
 module Moves
   class Finder
-    attr_accessor :filter_params, :ability
+    attr_reader :filter_params, :ability
 
-    def initialize(filter_params, ability = nil)
-      self.filter_params = filter_params
-      self.ability = ability
+    def initialize(filter_params, ability, order_params)
+      @filter_params = filter_params
+      @ability = ability
+      @order_by = (order_params[:by] || 'date').to_sym
+      @order_direction = if order_params[:by]
+                           (order_params[:direction] || 'asc').to_sym
+                         else
+        # default if no 'by' parameter is date descending
+                           :desc
+                         end
     end
 
     def call
-      apply_filters(Move).order('moves.id')
+      scope = apply_filters(Move)
+      case @order_by
+      when :name
+        scope.joins(person: :profiles).merge(Profile.ordered_by_name(@order_direction))
+      when :from_location
+        scope.joins(:from_location).merge(Location.ordered_by_title(@order_direction))
+      when :to_location
+        scope.joins(:to_location).merge(Location.ordered_by_title(@order_direction))
+      when :prison_transfer_reason
+        scope.left_outer_joins(:prison_transfer_reason).merge(PrisonTransferReason.ordered_by_title(@order_direction))
+      when :created_at
+        scope.order(created_at: @order_direction)
+      when :date_from
+        scope.order(date_from: @order_direction)
+      when :date
+        scope.order(date: @order_direction)
+      else
+        scope
+      end
```

Comment:
this method could be simplified a bit, 
maybe creating more private class with meaningful names,